<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderForm</class>
 <widget class="QMainWindow" name="OrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OrderForm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_order">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Order</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_rmk1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_reset">
    <property name="geometry">
     <rect>
      <x>161</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>212</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_dir">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_offset">
         <property name="text">
          <string>Offset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_strg">
         <property name="text">
          <string>Strategy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_exch">
         <property name="text">
          <string>Exchange</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_symbol">
         <property name="text">
          <string>Symbol</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_price">
         <property name="text">
          <string>Price</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_qty">
         <property name="text">
          <string>Qty</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QComboBox" name="cb_Dir"/>
       </item>
       <item>
        <widget class="QComboBox" name="cb_offset"/>
       </item>
       <item>
        <widget class="QComboBox" name="cb_strg"/>
       </item>
       <item>
        <widget class="QComboBox" name="cb_exch"/>
       </item>
       <item>
        <widget class="QLineEdit" name="le_symbol"/>
       </item>
       <item>
        <widget class="QLineEdit" name="le_price"/>
       </item>
       <item>
        <widget class="QLineEdit" name="le_qty"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="te_rmk2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_log">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>82</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_rmk1">
       <property name="text">
        <string>Remark1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_rmk2">
       <property name="text">
        <string>Remark2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_log">
       <property name="text">
        <string>Log Infor</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
